src/content/developers/docs/transactions/index.md

```diff
@@ -104,6 +104,35 @@ Example response:
 
 With the signature hash, the transaction can be cryptographically proven that it came from the sender and submitted to the network.
 
+### The data field {#the-data-field}
+
+The vast majority of transactions access a contract from an externally owned account (EOA).
+Most contracts are written in Solidity and interpret their data field in accordance with the [application binary interface (ABI)](https://docs.soliditylang.org/en/v0.8.12/abi-spec.html).
+
+The first four bytes specify which function to call, using the hash of the function's name and arguments.
+You can sometimes identify the function from the selector using [this database](https://www.4byte.directory/signatures/).
+
+The rest of the calldata is the arguments, [encoded as specified in the ABI specs](https://docs.soliditylang.org/en/v0.8.12/abi-spec.html#formal-specification-of-the-encoding).
```

Comment:
Is there a way to make this link timeless? It'll technically be out of date at every patch upgrade (especially a problem for non-English languages which aren't as frequently translated)